packages/serverless/src/gcpfunction/http.ts

```diff
@@ -0,0 +1,95 @@
+// '@google-cloud/functions-framework/build/src/functions' import is expected to be type-only so it's erased in the final .js file.
+// When TypeScript compiler is upgraded, use `import type` syntax to explicitly assert that we don't want to load a module here.
+import { HttpFunction } from '@google-cloud/functions-framework/build/src/functions';
+import { captureException, flush, getCurrentHub, Handlers, startTransaction, withScope } from '@sentry/node';
+import { logger, stripUrlQueryAndFragment } from '@sentry/utils';
+
+import { addServerlessEventProcessor, getActiveDomain, WrapperOptions } from './general';
+
+type Request = Parameters<HttpFunction>[0];
+type Response = Parameters<HttpFunction>[1];
+type ParseRequestOptions = Handlers.ParseRequestOptions;
+
+export interface HttpFunctionWrapperOptions extends WrapperOptions {
+  parseRequestOptions: ParseRequestOptions;
+}
+
+export { Request, Response };
+
+const { parseRequest } = Handlers;
+
+/**
+ * Capture exception with additional request information.
+ *
+ * @param e exception to be captured
+ * @param req incoming request
+ * @param options request capture options
+ */
+function captureRequestError(e: unknown, req: Request, options: ParseRequestOptions): void {
+  withScope(scope => {
```

Comment:
Missing `runtime` and `server_name`. Maybe it's worth extracting it into a generic function that can be reused in both `captureRequestError` and `captureEventError`?

Reply:
@kamilogorek 

`runtime` and `server_name` are being added by `parseRequest` directly to `event` object.

Reply:
Ah right, missed the fact that it's the same flow.